### PR TITLE
54 wordcloud backend communication

### PR DESCRIPTION
**Pull request by nano1342**

### CODE REVIEW

File: src-front/src/main.tsx [hunk 1]
```diff
@@ -15,6 +15,6 @@ import { router } from "./router.tsx";
 
 createRoot(document.getElementById("root")!).render(
   <StrictMode>
-    <RouterProvider router={router} />
+    <RouterProvider router={router} />,
```
Comment (Picorims) [inline src-front/src/main.tsx:18]: Are the commas normal?

File: src-front/src/components/WordCloud/WordCloud.tsx [hunk 2]
```diff
@@ -161,32 +101,214 @@ export default function WordCloud() {
...
+    });
+    session.onAcknowledgeLetterThen((accepted, currentState) => {
+      console.log("Letter acknowledged", accepted, currentState);
+    });
+    session.onAcknowledgeLetterErasedThen((accepted, currentState) => {
+      console.log("Letter erased acknowledged", accepted, currentState);
+    });
+    session.onAcknowledgeSubmitThen((success, testedState) => {
+      console.log("Submit acknowledged", success, testedState);
+      //réponse de si le mot a été réussi (valeur success)
```
Comment (Picorims) [inline src-front/src/components/WordCloud/WordCloud.tsx:173]: This should be in English.

**Review by Picorims — CHANGES_REQUESTED** [review]

File: src-front/src/components/WordCloud/WordCloud.tsx (hunk 2 above)
Comment (nano1342) [reply src-front/src/components/WordCloud/WordCloud.tsx]: Hey I just fixed it.

File: src-front/src/main.tsx (hunk 1 above)
Comment (nano1342) [reply src-front/src/main.tsx]: Hey I just fixed it.

**Review by Picorims — APPROVED** [review]

**Pull request merged**